docs/actors/development/base_docker_images.md

```diff
@@ -8,7 +8,15 @@ paths:
 
 # [](#base-docker-images) Base Docker images
 
-Apify provides the following Docker images that can be used as a base for user actors. You can read more about them in the [Apify SDK Docker image guide](https://sdk.apify.com/docs/guides/docker-images).
+Apify provides several Docker images that can be used as a base for user actors.
+
+All images come in two versions: the **latest** tag corresponds to the stable version and **beta** to images where we test new features. Use the beta version at your own risk.
+
+Note that all Apify Docker images are pre-cached on Apify servers in order to speed up the actor builds and runs. The source code used to generate the images is available in the [apify-actor-docker](https://github.com/apifytech/apify-actor-docker) GitHub repository.
```

Comment:
Not true ATM but we will fix this ASAP hopefully

Reply:
Yeah, this was just copied over from the old texts.